db/schema.rb

```diff
@@ -11,55 +11,53 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20_160_907_024_814) do
```

Comment:
I guess line below needs to be deleted and the line above is the one that should be included. The line above is the newer version. But i am not sure if that matters. @wdehaes